### ISSUE TITLE
Can't use source code

### ISSUE THREAD

**Issue by Chaotic0Mayhem** [turtle-insect/DQB2#13]
I downloaded the source code given, but it seems that it's set in debug mode and wants me to have the cmndat.exe file in bin\debug. But, when I downloaded your exe files, created the bin and debug folder and then copied the cmndat.exe into the folder it said that a "fatal error had occurred".

It should just allow me to build the project and not need me to download the exe files for it to do it's thing. Hopefully it can be fix and ty for the work you are doing.

**Comment by turtle-insect** [comment]
Most likely dependent on your environment.
Check the access permissions for the location of the source code
Check the solution platform when you build

**Comment by Chaotic0Mayhem** [comment]
Yeah, I found out that even if I downloaded .net 7.0 that it wouldn't fix my problem as it had in the past. I had planed to upgrade my Visual Studio from 2020 to 2022, but the files I wanted wouldn't be able to be installed 'til I got a bigger SSD and moved some of my games over to it.

As soon as I had installed Visual Studio 2022 and all the files I wanted. I tried your source code and it actually built it without the trouble of trying to go into a debug mode like it was before. TY for the help though.

Though I did run into another problem, but I don't think I need to open another issue as 1. it still fits with the issue as I can't use the built version and 2 I can just tell you here. When I built the project there were missing files from it. The files are "item.txt, story_island.txt, material_island.txt, mood.txt" and crop.txt" files. I had some of the files from the older version of this, but had to download the built version of this to get the rest.
